ext/pdo_firebird/tests/gh13119.phpt

```diff
@@ -0,0 +1,77 @@
+--TEST--
+GH-13119 (float, double value is incorrect)
+--EXTENSIONS--
+pdo_firebird
+--SKIPIF--
+<?php require('skipif.inc'); ?>
+--XLEAK--
+A bug in firebird causes a memory leak when calling `isc_attach_database()`.
+See https://github.com/FirebirdSQL/firebird/issues/7849
+--FILE--
+<?php
+
+require("testdb.inc");
+
+$dbh->exec('CREATE TABLE gh13119 (f_val FLOAT, d_val DOUBLE PRECISION)');
+
+$dbh->exec('INSERT INTO gh13119 VALUES (0.1, 0.1)');
+$dbh->exec('INSERT INTO gh13119 VALUES (0.0000000000000001, 0.0000000000000001)');
+$dbh->exec('INSERT INTO gh13119 VALUES (12.000000, 12.00000000000000)');
+$dbh->exec('INSERT INTO gh13119 VALUES (12.000001, 12.00000000000001)');
+$dbh->exec('INSERT INTO gh13119 VALUES (12.345678, 12.34567890123456)');
+$dbh->exec('INSERT INTO gh13119 VALUES (0.0000000000000000012345678, 0.000000000000000001234567890123456)');
+
+$stmt = $dbh->query('select * from gh13119');
+var_dump($stmt->fetchAll(PDO::FETCH_ASSOC));
+?>
+--CLEAN--
+<?php
+require 'testdb.inc';
+@$dbh->exec('DROP TABLE gh13119');
+unset($dbh);
+?>
+--EXPECT--
+array(6) {
+  [0]=>
+  array(2) {
+    ["F_VAL"]=>
+    string(3) "0.1"
+    ["D_VAL"]=>
+    string(3) "0.1"
+  }
+  [1]=>
+  array(2) {
+    ["F_VAL"]=>
+    string(7) "1.0E-16"
```

Comment:
I wonder whether the use of scientific notation breaks BC somehow though, although in arithmetic they might not as they're numeric strings.

Reply:
We can do the calculations, so I think it's okay.

https://3v4l.org/DL0bk